packages/components/src/test/properties/NavigationPropertyTargetSelector.test.tsx

```diff
@@ -121,11 +122,11 @@ describe("NavigationPropertyTargetSelector", () => {
     expect((ref.current?.getValue() as PrimitiveValue).value).to.be.undefined;
 
     const select = await waitFor(() => {
-      const element = container.querySelector<HTMLDivElement>(".iui-input-with-icon");
+      const element = container.querySelector<HTMLDivElement>(".iui-select-button");
```

Comment:
Maybe we can use `getByRole("input")` to open dropdown menu?

Reply:
changed to `getByRole("textbox")`